data/data-pipeline/data_pipeline/etl/score/etl_score.py

```diff
@@ -310,61 +316,21 @@ def _add_percentiles_to_df(
 
         else:
             # For agricultural loss, we are using whether there is value at all to determine percentile and then
```

Comment:
Is this comment now misplaced because it refers to agricultural loss?

Reply:
No, because agricultural loss is one of the two cases covered here. Does that make sense to you? I can adjust the comment to make it clearer

Reply:
Yeah it seems like some adjustment could be useful. It also may not be necessary to mention the specific use cases *inside* this method. Because you've abstracted this all to `drop_tracts` I don't think you need to mention the use cases anymore.

Reply:
Updated!